src/main/target/STM32F7X2/config/NERO.config

```diff
@@ -0,0 +1,79 @@
+// Usual lawyer stuff here
+
+# Basic I/O
```

Comment:
I think we should do a `defaults nosave` at the beginning of this, so that users can copy / paste it and get reproducible results.

Reply:
Good thinking...